modules/custom/cu_redirect/cu_redirect.module

```diff
@@ -20,6 +20,39 @@ function cu_redirect_menu() {
   return $items;
 }
 
+/**
```

Comment:
We don't want more secure_permission hooks.  Any module that's a core or profile dependency should add the permissions to express_permissions.  It's only bundles that need to add this hook.